rector_examples_updated/entity_type_getsingularlabel.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * EntityType::getSingularLabel() now handles lower-casing the entity type label; getLowercaseLabel() deprecated.
+ */
+
+/**
+ * A simple example.
+ */
+function simple_example() {
+  /* @var \Drupal\node\Entity\Node $node */
+  $node = \Drupal::entityTypeManager()->getStorage('node')->load(123);
+  $entity_type = $node->getEntityType();
+  $entity_type->getSingularLabel();
+}
```

Comment:
These is failing the automated test, because this "after" file needs to match exactly what Drupal Rector produces. I usually copy and past the "before" and then run Rector on it.

Can you update this one that way?

Reply:
@damontgomery The tests are passing now.